src/asynqp/__init__.py

```diff
@@ -48,7 +48,13 @@ def connect(host='localhost',
         kwargs['port'] = port
 
     dispatcher = Dispatcher()
-    transport, protocol = yield from loop.create_connection(lambda: AMQP(dispatcher, loop), **kwargs)
+    try:
+        transport, protocol = yield from loop.create_connection(lambda: AMQP(dispatcher, loop), **kwargs)
+    except (ConnectionRefusedError, OSError):
+        # Throw a single exception instead of two
+        raise ConnectionRefusedError('Failed to connect - host: {} port: {}'
+                                     ' username: {} password: {} virtual_host: {}'
+                                     .format(host, port, username, password, virtual_host))
```

Comment:
It depends how you look at it. When this method fails, python raises both a ConnectionRefuseError and an OsError - MultiError:

I found it a bit of a pain in the client code to have to remember to capture both errors to retry a connection, it's much more intuitive to catch ConnectionError and retry.

The from e is a good thing, I'll add it.